pyramid_zipkin/logging_helper.py

```diff
@@ -11,8 +11,17 @@
 from pyramid_zipkin.thrift_helper import create_span
 from pyramid_zipkin.thrift_helper import thrift_obj_in_bytes
 
+try:  # Python 2.7+
```

Comment:
This is simple, so we can leave it, but in general we err on the side of dropping py26 support rather than falling over ourselves to maintain it.